api/server.go

```diff
@@ -77,12 +77,19 @@ func Serve(ctx context.Context, serveAddr string, db idb.IndexerDb, fetcherError
 		middleware = append(middleware, middlewares.MakeAuth("X-Indexer-API-Token", options.Tokens))
 	}
 
+	swag, err := generated.GetSwagger()
```

Comment:
Technically there is also `common.GetSwagger()`, but I don't think the Health endpoint has any parameters.

I only bring it up because you included the boilerplate code on MakeHealthCheck but there's no way for it to be part of the DisableMap.

Reply:
MakeHealthCheck doesn't have the checking code...will it ever need it?

Reply:
No, we can ignore it there